spec/models/book_spec.rb

```diff
@@ -10,9 +10,26 @@
     expect(book.subjects_as_tuples).to eq [["foo", 1]]
   end
 
+  it "can list binding as tuples (hashes) of (:format,:price,:ean,:status,:pub_date)" do
+    book = described_class.new
+    book.assign_attributes("bindings" => "{\"binding\":[{\"format\":\"PB\",\"price\":\"$31.95\",\"ean\":\"978-1-59213-386-4\",\"binding_status\":\"IP\",\"pub_date_for_format\":\"Jun 06\"}]}")
+    expect(book.bindings_as_tuples).to eq [{ format: "PB", price: "$31.95", ean: "978-1-59213-386-4", status: "IP", pub_date: "Jun 06" }]
+  end
+
+  it "can add sort title to book before save" do
+    book = described_class.new
+    book.assign_attributes(title: "The Way to Nirvana")
+
+    expect(book.sort_titles).to eq "Way to Nirvana, The"
+  end
+
+  it_behaves_like "detachable"
```

Comment:
I haven't seen this before.  The two lines seems to contradict each other.  It should behave as both detachable and attachable?  What does that mean?